backend/src/main/java/com/beaplab/BeaplabEngine/config/RollbarConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.beaplab.BeaplabEngine.config;
+
+import static com.rollbar.notifier.config.ConfigBuilder.withAccessToken;
+import com.rollbar.notifier.Rollbar;
+import com.rollbar.notifier.config.Config;
+
```

Comment:
Specify a property source here so we can store our vars. Take a look at RedisConfig for reference
`@PropertySource("classpath:rollbar.properties")`

Basically java spring will find a `rollbar.properties` file inside src/main/resources

Then inside `rollbar.properties` file

You can define it like this

access.token = ${ROLLBAR_ACCESS_TOKEN}

What this means is when the application is running it will assign the environment variable `ROLLBAR_ACCESS_TOKEN` to access.token in the properties

Reply:
If you want to test that its working you can create a .env file in the root directory and assign the rollbar access token there, and dont worry it wont be committed and will only be local to you.

`ROLLBAR_ACCESS_TOKEN=yourtokenhere`

Reply:
Once you are confident that it works could you add this line in digitalocean_deploy.yml?

After line 19 which is `echo "PG_CRYPTO_KEY=${{ secrets.PG_CRYPTO_KEY }}" >> .env`

Add this if it is possible to reuse the same token we use for the react app you can just add
`echo "ROLLBAR_ACCESS_TOKEN=${{ secrets.ROLLBAR_ACCESS_TOKEN}}" >> .env`

Or if we can't then we can just define a new value to something like this and ardalan can add the secrets for it later
`echo "ROLLBAR_ACCESS_TOKEN=${{ secrets.JAVA_ROLLBAR_ACCESS_TOKEN}}" >> .env`